credentials/tls/certprovider/meshca/go.mod

```diff
@@ -0,0 +1,10 @@
+module google.golang.org/grpc/credentials/tls/certprovider/meshca
+
+go 1.14
+
+require (
+	github.com/envoyproxy/go-control-plane v0.9.5-0.20200326174812-e8bd2869ff56
+	github.com/golang/protobuf v1.4.2
+	google.golang.org/grpc v1.31.0-dev.0.20200616205733-dfc058c6d9dd
+	istio.io/istio v0.0.0-20200624211210-fba43d664ca1
```

Comment:
Why is this not using a release?

Reply:
Strange. Maybe they don't do releases with a `git tag`?

```
easwars@easwars-macbookpro:meshca $ go list -m -json istio.io/istio@latest
{
        "Path": "istio.io/istio",
        "Version": "v0.0.0-20200625184358-746a2b4c58c7",
        "Time": "2020-06-25T18:43:58Z",
        "Dir": "/Users/easwars/go/pkg/mod/istio.io/istio@v0.0.0-20200625184358-746a2b4c58c7",
        "GoMod": "/Users/easwars/go/pkg/mod/cache/download/istio.io/istio/@v/v0.0.0-20200625184358-746a2b4c58c7.mod",
        "GoVersion": "1.13"
}
```

Reply:
Maybe because their tags are like `1.4.10` ([ref](https://github.com/istio/istio/releases/tag/1.4.10)) instead of `v1.4.10` ([our latest release](https://github.com/grpc/grpc-go/releases/tag/v1.30.0))?